pages/index.js

```diff
@@ -24,75 +25,95 @@ export default function Home() {
 				<meta name="twitter:image" content="https://hackropolis.club/logo.png" />
 				<link rel="canonical" href="https://hackropolis.club/" />
 			</Head>
-			<main className={styles.main}>
+			<div className={styles.header}>
 				<h1 className={styles.title}>
 					We are <u>Hackropolis</u>
 				</h1>
-				<h2 className={styles.slogan}>The first virtual hackerspace in Greece</h2>
-				<p className={styles.description}>
-					<b>Let's be honest: making stuff alone kinda sucks.</b> <br />
-					We're a virtual club (member of{' '}
-					<a
-						href="https://hackclub.com/?utm_source=hackropolis.vercel.app&utm_medium=linkback&utm_campaign=hackropolis"
-						target="_blank"
-						rel="noopener"
-					>
-						The Hack Foundation
-					</a>
-					), where like-minded students from Greece (<i>and beyond</i>) meet weekly to discuss about topics they are interested in, build
-					stuff they like and learn together.
-				</p>
+				<h2 className={styles.slogan}>The first virtual computer club for youths in Greece</h2>
+			</div>
+			<main className={styles.main}>
+				<article className={styles.description}>
+					<p>
+						<b>Are you interested in computer science?</b>
+					</p>
+					<p>
+						Hackropolis is a virtual computer club for Greek and/or Greek-speaking young people that are interested in computer science
+						and other related fields. We regularly organize meetings online and work on projects together. No experience required — all
+						you need is an internet connection, come as you are!
+					</p>
+					<p>
+						We mostly consist of middle-to-high school students and operate under the auspices of Hack Club (which is a registered
```

Comment:
`hack club` => `The Hack Foundation` and link back to the site with the url from above